components/blog-article.tpl

```diff
@@ -8,7 +8,9 @@
       {% endif %}
     </h1>
 
-    <time class="article__date" datetime="{{ article.created_at | date: '%Y-%m-%d' }}">{{ article.created_at | format_date: "long" }}</time>
+    {% assign current_year = 'now' | date: "%Y" | to_num %}
```

Comment:
**compontents/blog-article.tpl** is a component that is used in the for loop in the **layouts/blog___news.tpl** layout (**line 27**).

There's no need to **assign** the `current_year` variable in that loop. To prevent reassigning that variable in each for loop iteration, move it out from the loop to the **layoyts/blog___news.tpl** into the `<main class="articles-container">` right before the loop.